docs/get-started/backup/on_demand_backups.md

```diff
@@ -0,0 +1,158 @@
+# Working with On-demand backups
+
+## [Make a backup](#make-a-backup)
+
+To create a backup:
+
+1. Go to <i class="uil uil-history"></i> **Backup > All Backups**.
+2. Click <i class="uil uil-plus-square"></i> **Create Backup**.
+3. Specify the type of backup that you want to create: **On Demand** or **Schedule Backup**.
+4. Enter a unique name for this backup.
```

Comment:
We're using the same steps for mysql and mongo. If we want to change something we need to do it in both files. I think we can restructure the docs. Probably we can import single file in several places.